gqueries/general/primary_demand/total_primary_demand_aggregated_carriers/aggregated_total_primary_demand_of_solar_thermal.gql

```diff
@@ -0,0 +1,5 @@
+- unit = PJ
+- query = 
+    SUM(
+      Q(total_primary_demand_of_solar_thermal)
```

Comment:
@kaskranenburgQ I'm not sure whether this works completely. The underlying query has the following structure:

```
- query =
    SUM(
      V(G(primary_energy_demand),primary_demand_of_solar_thermal)
    ) / BILLIONS
- unit = PJ
```

However, when I query the `energy_heat_solar_ht_solar_thermal` with `primary_demand_of_solar_thermal`, the demand is 0 (even when I've installed HT solar thermal capacity). The `primary_demand_of_steam_hot_water` is nonzero though.

In short, for `buildings_final_demand_solar_thermal` and `households_final_demand_solar_thermal`, `primary_demand_of_solar_thermal` works. For `energy_heat_solar_lt_solar_thermal`, `energy_heat_solar_mt_solar_thermal` and `energy_heat_solar_ht_solar_thermal`, `primary_demand_of_steam_hot_water` should be used.

I guess this impacts both the solar thermal and the imported heat queries?

Reply:
Implemented this